doc/templates.rst

```diff
@@ -76,8 +76,18 @@ Twig templates have access to variables provided by the PHP application and
 variables created in templates via the :doc:`set <tags/set>` tag. These
 variables can be manipulated and displayed in the template.
 
-Use a dot (``.``) to access attributes of a variable (methods or properties of a
-PHP object, or items of a PHP array):
+Twig tries to abstract PHP types as much as possible and works with a few basic
+types, supported by ``filters``, ``functions``, and ``tests`` among others:
+
+* ``string``: A sequence of characters;
+* ``number``: An integer or a float;
```

Comment:
should we distinguish integer and floats (or maybe integers and numbers, with integers being a subtype) ? We have a bunch of places expecting an integer (`cycle` for instance)

Reply:
I would just cast to an integer when needed